### PR TITLE
Add working links to the other pages on the homepage #87

### PR DESCRIPTION
**Pull request by anikaSchwing**
Also added view and feature test

### CODE REVIEW

File: app/views/pages/homepage.html.erb [hunk 1]
```diff
@@ -3,9 +3,9 @@
     <img src="" />
     <h1>Taste of Code</h1>
     <h3>Beginner Coding Workshop</h3>
-    <a href="#" class="btn btn-lg btn-secondary">Join TOC</a>
-    <a href="#" class="btn btn-lg btn-secondary">InCompany</a>
-    <a href="#" class="btn btn-lg btn-secondary">Host TOC</a>
+    <%= link_to "Join TOC", attend_path, class: "btn btn-lg btn-secondary" %>
+    <%= link_to "InCompany", incompany_path, class: "btn btn-lg btn-secondary" %>
+    <%= link_to "Host TOC", sponsor_path, class: "btn btn-lg btn-secondary" %>
   </div>
```
Comment (leefreemanxyz) [inline app/views/pages/homepage.html.erb]: I think this blank line can go.

File: spec/features/homepage_links_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'navigation from homepage', type: :feature do
+  describe 'testing links' do
+    it 'goes to the attend page when the Join TOC link is clicked' do
+      visit root_path
+      click_link('Join TOC')
+      expect(current_path).to eq(attend_path)
```
Comment (arnoFleming) [inline spec/features/homepage_links_spec.rb]: nice

File: spec/views/pages/home.html.erb_spec.rb [hunk 3]
```diff
@@ -1,5 +0,0 @@
-require 'rails_helper'
```
Comment (arnoFleming) [inline spec/views/pages/home.html.erb_spec.rb]: ty

File: spec/views/pages/homepage.html.erb_spec.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "pages/homepage.html.erb", type: :view do
+
```
Comment (arnoFleming) [inline spec/views/pages/homepage.html.erb_spec.rb]: you can remove this empty line here, and on line 21, like the spacing between the specs (line 10, 15) a lot!

Comment (arnoFleming) [inline spec/views/pages/homepage.html.erb_spec.rb]: and, btw, with that beautiful integration test, you could skip this unit test (but you don't have to)

File: app/views/pages/homepage.html.erb (hunk 1 above)
Comment (arnoFleming) [inline app/views/pages/homepage.html.erb]: 👍

**Pull request merged**